lib/publicConfig.ts

```diff
@@ -11,22 +11,50 @@ if (!canaryNetUrl) throw "Missing NEXT_PUBLIC_CANARY_NET_URL"
 const tokenAmountFlow = process.env.NEXT_PUBLIC_TOKEN_AMOUNT_FLOW
 if (!tokenAmountFlow) throw "Missing NEXT_PUBLIC_TOKEN_AMOUNT_FLOW"
 
+const tokenAmountFusd = process.env.NEXT_PUBLIC_TOKEN_AMOUNT_FUSD
+if (!tokenAmountFusd) throw "Missing NEXT_PUBLIC_TOKEN_AMOUNT_FUSD"
+
+const signerAddress = process.env.NEXT_PUBLIC_SIGNER_ADDRESS
+if (!signerAddress) throw "Missing NEXT_PUBLIC_SIGNER_ADDRESS"
+
+const walletDiscovery = process.env.NEXT_PUBLIC_WALLET_DISCOVERY
+// if (!walletDiscovery) throw "Missing NEXT_PUBLIC_WALLET_DISCOVERY"
```

Comment:
```suggestion
// TODO: Integrate FCL wallets
// if (!walletDiscovery) throw "Missing NEXT_PUBLIC_WALLET_DISCOVERY"
```

It's good to track these kinds of commented code snippets with TODOs